utils/cliutils/commandsflags.go

```diff
@@ -1235,6 +1233,10 @@ var flagsMap = map[string]cli.Flag{
 		Name:  PeriodicDBSyncV3,
 		Usage: fmt.Sprintf("[Default: false] Set to true to get the Xray DBSync V3 Periodic Package (Use with %s flag). ` `", DBSyncV3),
 	},
+	useWrapperAudit: cli.BoolTFlag{
+		Name:  UseWrapper,
+		Usage: "[Default: true] Set to false if you wish to not use the wrapper. ` `",
```

Comment:
Since this option is part of the "jf audit" command, which doesn't necessarily has a package manager context, I suggest changing - 
```
Set to false if you wish to not use the wrapper. 
```
intio - 
```
Set to false if you wish to not use the gradle or maven wrapper. 
```

Reply:
Done!